tests/test_sort.py

```diff
@@ -16,6 +17,10 @@
 
 
 class TestSuite(unittest.TestCase):
+    def test_bogo_sort(self):
+        self.assertEqual([1, 5, 23, 57, 65, 1232],
```

Comment:
I think we should use a small test case here (to reduce testing time) since it runs `O(n * n!)`. We should add a note in the description too, to avoid giving array with length > 6 (or 7 ?). It could take a lot of time for large `n`.